src/views/SignupLedger.vue

```diff
@@ -69,7 +68,7 @@ export default class SignupLedger extends Vue {
 
     private get loaderTitle() {
         return this.state !== SignupLedger.State.FINISHED
-            ? 'Fetching Accounts'
+            ? 'Fetching Account'
```

Comment:
I guess this is supposed to be `Fetching Addresses`?

Reply:
Do you like that more?
It's either fetching the whole account or fetching the addresses for that account.
Both are valid.

Reply:
I mean, because what was `accounts` before is now `addresses`, so to simple _translate_, it would be `Fetching Adresses`. But I have no opinion on what's _better_.